elixir/apps/web/lib/web/live/settings/api_clients/beta.ex

```diff
@@ -25,7 +25,7 @@ defmodule Web.Settings.ApiClients.Beta do
       <:title><%= @page_title %></:title>
       <:help>
         API Clients are used to manage Firezone configuration through a REST API. See our
-        <a class={link_style()} href={url(API.Endpoint, ~p"/swaggerui")}>interactive API docs</a>
+        <a class={link_style()} href="api.firezone.dev/swaggerui">interactive API docs</a>
```

Comment:
I think you need the scheme here otherwise it will be treated as a path

```suggestion
        <a class={link_style()} href="https://api.firezone.dev/swaggerui">interactive API docs</a>
```